examples/language/lib/classes/shorthand.dart

```diff
@@ -0,0 +1,172 @@
+// #docregion intro
+// Enum example
+enum Status { none, running, stopped, paused }
+Status currentStatus = .running; // Instead of Status.running
+
+// Static method example
+int port = .parse('8080'); // Instead of int.parse('8080')
+
+// Constructor example
+class Point {
+  final int x, y;
+  Point(this.x, this.y);
+  Point.origin() : x = 0, y = 0;
+}
+Point origin = .origin(); // Instead of Point.origin()
+// #enddocregion intro
+
+// #docregion enums
+// Enum example
+enum LogLevel { debug, info, warning, error }
+
+// Function to get a color code based on log level
+String colorCode(LogLevel level) {
+  return switch (level) {
+    .debug   => 'gray',    // Instead of LogLevel.debug
+    .info    => 'blue',    // Instead of LogLevel.info
+    .warning => 'orange',  // Instead of LogLevel.warning
+    .error   => 'red',     // Instead of LogLevel.error
+  };
+}
+
+// Example usage:
+var warnColor = colorCode(.warning); // Returns 'orange'
+
+// #enddocregion enums
+
+// #docregion methods
+// Static method
+int httpPort = .parse('80'); // Instead of int.parse('80')
+
+// Static field/getter
+BigInt bigIntZero = .zero; // Instead of BigInt.zero
+// #enddocregion methods
+ 
+// #docregion constructors
+ class Point {
```

Comment:
Since this file already has a `Point` class, it can't have a second one (or the third one later on). Consider splitting some of these examples to more focused files. Or I can do so for you if you prefer, just let me know!